remkus/genesis-translations
# Add translations for Genesis 3.1, including JavaScript strings

Uses the build process outlined in https://github.com/remkus/genesis-translations/pull/10 to generate Genesis 3.1 language files, and is based on that PR.

To test:

1. Activate Genesis Translations.
2. Install Genesis 3.1.
3. Confirm that Genesis translations are picked up by WordPress when switching languages. You should see Spanish strings in the new Genesis sidebar when switching to Spanish.

Notes:

- I have not bumped the plugin version, edited the changelog or readme, or performed any other release tasks in this PR.
- The `wp i18n make-json` command that the build process uses currently generates a JSON file for every language file, even if those files do not contain entries for JavaScript translations. Some of the JSON files therefore contain empty translations, but English is still used as the fallback in those cases.
